setup.py

```diff
@@ -14,8 +14,7 @@
           'backoff==1.8.0',
           'requests==2.20.0',
           'python-dateutil==2.6.0',
-          'PureCloudPlatformApiSdk==0.45.1.101',
-          'PureCloudPlatformClientV2==26.0.0',
+          'PureCloudPlatformClientV2==64.0.1',
```

Comment:
Removed PureCloudPlatformApiSdk since it hasn't been updated in 5+ years.

No particular reason to stop at 64.0.1. I chose this since `get_client_credentials_token` was introduced in 57.0.1 and last updated in 64.0.1 https://github.com/MyPureCloud/platform-client-sdk-python/blame/master/build/PureCloudPlatformClientV2/api_client.py#L104-L132

I can probably keep upgrading until something breaks.

Reply:
Bumped to latest version 151.0.0 without any issues.